samples/boards/stm32/power_mgmt/blinky/prj.conf

```diff
@@ -2,3 +2,5 @@ CONFIG_PM=y
 CONFIG_PM_DEVICE=y
 CONFIG_PM_DEVICE_RUNTIME=y
 CONFIG_PM_DEVICE_RUNTIME_EXCLUSIVE=n
+# Increase IDLE stack for the IDLE timer
```

Comment:
Please clarify that this is valid only for non-LPTIM implementation (or add a boards/nucleo_f429zi.conf file)

Reply:
Done.